src/main/java/seedu/address/logic/parser/tag/DeleteTagCommandParser.java

```diff
@@ -56,10 +58,10 @@ public DeleteTagCommand parse(String args) throws ParseException {
         parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).ifPresent(editTaskDescriptor::setTags);
 
         if (!editPersonDescriptor.isAnyFieldEdited()) {
```

Comment:
should it be (!editPersonDescriptor.isAnyFieldEdited() && !editTaskDescriptor.isAnyFieldEdited())?